pallets/omnipool/src/lib.rs

```diff
@@ -359,6 +363,12 @@ pub mod pallet {
 
 		/// Asset's weight cap has been updated.
 		AssetWeightCapUpdated { asset_id: T::AssetId, cap: Permill },
+
+		/// Amount of the Hub asset has been updated.
+		HubAmountUpdated {
+			hub_amount_in: Balance,
+			hub_amount_out: Balance,
```

Comment:
batch-id? (I would probably rename it to event-id or something similar

Reply:
why does this event `HubAmountUpdated` need the batch/event id?